### PR TITLE
Update FFDC cleanup messages for translation

### PR DESCRIPTION
**Pull request by tonyreigns**
Updating the hardcoded FFDC cleanup message in order to be translated.

### CODE REVIEW

**Comment by tonyreigns** [comment]
#build

File: dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops [hunk 1]
```diff
@@ -168,6 +168,10 @@ JSON_ACCESS_LOG_NO_HEADER_NAME_SPECIFIED=TRAS3012W: Header name is missing for o
 JSON_ACCESS_LOG_NO_HEADER_NAME_SPECIFIED.explanation=JSON logging uses the logFormat attribute from accessLogging to select which http_access.log message fields to include. A header name must be supplied when using the %i or %o fields in the accessLogging logFormat attribute.
 JSON_ACCESS_LOG_NO_HEADER_NAME_SPECIFIED.useraction=Provide a header name for the field in the accessLogging logFormat attribute.
 
+FFDC_FILE_DELETION=TRAS3014I: Deleted {0} FFDC file(s) based on the maxFfdcAge value configured.
```
Comment (helyarp) [inline dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops]: Don't use (s) and move configured -> "Deleted {0} FFDC files based on the configured maxFfdcAge value."

**Review by helyarp — COMMENTED** [review]
Small changes suggested

File: dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops (hunk 1 above)
Comment (tonyreigns) [reply dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops]: The message has been fixed, thanks.

**Review by helyarp — COMMENTED** [review]
Looks good!

**Review by fmhwong — APPROVED** [review]
LGTM

**Comment by tonyreigns** [comment]
#libby

**Pull request merged**